### PR TITLE
Fixing tests for new IIF

### PR DESCRIPTION
**Pull request by GearsAD**
The downstream tests are borked - this should fix them.

### CODE REVIEW

File: test/iifCompareTests.jl [hunk 1]
```diff
@@ -56,7 +56,7 @@ using Test
     # fg2 has been solved, so it should fail on the estimate dictionary
     @test !compareSimilarVariables(fg, fg2, skipsamples=true, skip=Symbol[:initialized;:inferdim])
 
-    tree = wipeBuildNewTree!(fg2)
+    tree = solveGraph!(fg2)
```
Comment (dehann) [inline test/iifCompareTests.jl:59]: you could also just build tree with resetBuildTree!(fg)

**Review by dehann — APPROVED** [review]
one comment if you wanted, but looks good!

File: test/iifCompareTests.jl (hunk 1 above)
Comment (GearsAD) [reply test/iifCompareTests.jl:59]: yeah I couldn't find out what the new name was - thanks!

Comment (dehann) [reply test/iifCompareTests.jl:59]: also the return from 'tree, smt, hist = solveTree!(fg)' is a Tuple

Comment (dehann) [reply test/iifCompareTests.jl:59]: maybe we should consider a better name 'buildTreeReset!(fg)' which is more probable and better fits verbNouns

**Review by Affie — APPROVED** [review]

**Pull request merged**